### PR TITLE
Python Script Updates

### PR DESCRIPTION
**Pull request by mark-lunarg**
Removed an unused script and changed the default output mode for doc validator tests

### CODE REVIEW

File: tests/_vkvalidatelayerdoc.ps1 [hunk 1]
```diff
@@ -18,9 +18,7 @@ if ($args[0] -eq "-Debug") {
     $dPath = "Release"
 }
 
-if (($args[0] -eq "terse_mode") -Or ($args[1] -eq "terse_mode")) {
-    $output_mode = "terse_mode"
-}
+$output_mode = "terse_mode"
```
Comment (jzulauf-lunarg) [inline tests/_vkvalidatelayerdoc.ps1]: Shouldn't we replace the arg 0/1 check with "verbose_mode" and set terse unless verbose is found?

File: tests/vkvalidatelayerdoc.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ printf "$GREEN[ RUN      ]$NC $0\n"
 pushd ../../scripts
 
 # Validate that layer database matches source contents
-python3 vk_validation_stats.py $1
+python3 vk_validation_stats.py terse_mode
```
Comment (jzulauf-lunarg) [inline tests/vkvalidatelayerdoc.sh]: same thing here.  look for -v or --vebose_mode as $a and pass on terse or not as appropriate.

**Comment by mark-lunarg** [comment]
Changed logic over to use a 'verbose' arg.

File: tests/vkvalidatelayerdoc.sh [hunk 3]
```diff
@@ -24,7 +24,7 @@ printf "$GREEN[ RUN      ]$NC $0\n"
 pushd ../../scripts
 
 # Validate that layer database matches source contents
-python3 vk_validation_stats.py $1
+python3 vk_validation_stats.py
```
Comment (jzulauf-lunarg) [inline tests/vkvalidatelayerdoc.sh]: We've still lost the ability to control the output mode in this script.

File: tests/_vkvalidatelayerdoc.ps1 [hunk 4]
```diff
@@ -18,18 +18,14 @@ if ($args[0] -eq "-Debug") {
...
-
 write-host -background black -foreground green "[  RUN     ] " -nonewline
 write-host "vkvalidatelayerdoc.ps1: Validate layer documentation"
 
 # Run doc validation from project scripts dir
 push-location ..\..\scripts
 
 # Validate that layer documentation matches source contents
-python vk_validation_stats.py $output_mode
+python vk_validation_stats.py
```
Comment (jzulauf-lunarg) [inline tests/_vkvalidatelayerdoc.ps1]: We used to be able to control the output mode, but we've lost that here.

Comment (mark-lunarg) [reply tests/_vkvalidatelayerdoc.ps1]: True, and having the verbose output is an unnecessary use-case for these scripts.  The verbose output is still available by running the doc validator directly.

**Pull request merged**